img/cms/.htaccess

```diff
@@ -2,7 +2,7 @@
 	php_flag engine off
 </IfModule>
 deny from all
-<Files ~ "(?i)^.*\.(jpg|jpeg|gif|png|bmp|tiff|svg|pdf|mov|mpeg|mp4|avi|mpg|wma|flv|webm)$">
+<Files ~ "(?i).*\.(jpg|jpeg|gif|png|bmp|tiff|svg|pdf|mov|mpeg|mp4|avi|mpg|wma|flv|webm)$">
```

Comment:
An expression still needs the caret character at the beginning of the regular expression search. IMHO I think this breaks.